pom.xml

```diff
@@ -71,6 +71,23 @@
   <build>
     <plugins>
       <plugin>
+        <groupId>org.commonjava.maven.plugins</groupId>
+        <artifactId>directory-maven-plugin</artifactId>
+        <version>0.1</version>
```

Comment:
There is a version 0.2 available (see https://mvnrepository.com/artifact/org.commonjava.maven.plugins/directory-maven-plugin) - any special reason why you nonetheless chose 0.1?

Reply:
No special reason, I use the version 0.1 in several projects and it is doing its job.
It seems the version 0.2 has been released on July 31th, 2017 and I was not aware of.
If 0.2 is working as expected, let's use the newer one.